main.py

```diff
@@ -303,21 +320,23 @@ def quantize_aq(
             out_losses = update_outs_parallel(
                 args.devices, layer, inps, outs, compute_mse=not args.skip_out_loss, **forward_args
             )
+        stats_payload["out_loss"] = torch.mean(torch.Tensor(out_losses)).item()
 
         if run_validation:
+            should_compute_mse = not (args.skip_out_loss or loaded_layer)
```

Comment:
Rationale: if we loaded a layer, we do not compute outs before quantization (so as to save time)